pkg/api/v20191231preview/openshiftcluster_validatestatic.go

```diff
@@ -41,7 +42,7 @@ func (sv *openShiftClusterStaticValidator) Static(_oc interface{}, _current *api
 	var err error
 	sv.r, err = azure.ParseResourceID(sv.resourceID)
 	if err != nil {
-		return err
+		return errors.WithStack(err)
```

Comment:
> v20191231preview

Should we be changing this version? Isn't it frozen?

Reply:
the API is not changing just the content of an error.